starter-code/src/viking.js

```diff
@@ -1,11 +1,76 @@
-// Soldier
-function Soldier () {}
+function Soldier (health, strength) {
+  this.health = health;
+  this.strength = strength;
+
+  this.receiveDamage = function (damage){
+    this.health = this.health - damage;
+  };
+}
+
+Soldier.prototype.attack = function (){
+   return this.strength;
+  };
+
+
+  // Viking
+
+function Viking (name, health, strength) {
+  this.name = name;
+  this.health = health;
+  this.strength = strength;
+
+  this.receiveDamage = function (damage){
+    this.health = this.health - damage;
+    if (this.health > 0) {
+      return this.name + " has received " + damage + " points of damage";
+    }
+    else {
+      return this.name + " has died in act of combat";
+    }
+  };
+  this.battleCry = function (){
+    return  "Odin Owns You All!";
+  };
+ }
+
+Viking.prototype = Object.create(Soldier.prototype);
+
```

Comment:
Por otro lado, faltaria cambiar el constructor de Vikigo con un 
Viking.prototype.constructor = Viking.

De esta manera puedes acceder a la propiedad exclusiva de Viking (this.name = name), además de aquellas a las que accedes gracias a la herencia de Soldier.